packages/block-library/src/cover/edit.js

```diff
@@ -397,6 +397,15 @@ function CoverEdit( {
 	const isImageBackground = IMAGE_BACKGROUND_TYPE === backgroundType;
 	const isVideoBackground = VIDEO_BACKGROUND_TYPE === backgroundType;
 
+	useEffect( () => {
```

Comment:
I tried an alternative approach here of checking to see if url was set instead and it seemed to work for image, video, plain color background, and for transforming from image. This also has the advantage of not needing to add deprecation so I wonder it that is a better way to fix this?